src/S3/Crypto/S3EncryptionMultipartUploaderV2.php

```diff
@@ -105,6 +108,7 @@ public function __construct(
         $source,
         array $config = []
     ) {
+        $this->appendUserAgent($client, 'S3CryptoV2.0');
```

Comment:
At some point, have we considered a global file for these magic strings that are repeated across files like "S3CryptoV2.0".  There's so many different ones that I feel like implementing that would be messier and may be more trouble than it's worth, I'm just curious if the idea has been floated before